doc/rest_api/snippet_sharing/use_cases/UC_edit_user.txt

```diff
@@ -0,0 +1,17 @@
+Summary:
+----------------------------
+Title: edit user
+Scope: User Management
+Level: User Goal
+Actors: Authenticated user, Administrator
+Brief: Users can edit their own account information the same way Administrators can do it for them. Administrators can also change permissions of users.

```

Comment:
"the same way" suggests that the other way was explained somewhere else. Please link to use case or change the sentence.

Do you also plan some automatical scoring for uploading many popular snippets?